src/components/ZetkinRelativeTime.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import ZetkinRelativeTime from './ZetkinRelativeTime';
+
+export default {
+  component: ZetkinRelativeTime,
+  title: 'Atoms/ZetkinRelativeTime',
+} as ComponentMeta<typeof ZetkinRelativeTime>;
+
+const Template: ComponentStory<typeof ZetkinRelativeTime> = (args) => (
+  <ZetkinRelativeTime
+    convertToLocal={args.convertToLocal}
+    datetime={args.datetime}
+    forcePast={args.forcePast}
+  />
+);
+
+export const basic = Template.bind({});
+basic.args = {
+  datetime: '2022-06-01T14:53:15',
+};
+
+export const toLocal = Template.bind({});
+toLocal.args = {
+  convertToLocal: true,
+  datetime: '2022-06-01T14:53:15',
+};
```

Comment:
Might be really nice to have a little documentation here about how and why this is used, or what impact it has. Storybook can generate some interactive docs im pretty sure.

Reply:
it seems in their docs that you're supposed to just be able to put comments in the stories-file and they will render in the docs tab in storybook, but I'm not getting it to work for me :(

Reply:
I'll look in to this!